easybuild/easyconfigs/h/HDF5/HDF5-1.8.19-foss-2017a.eb

```diff
@@ -0,0 +1,21 @@
+name = 'HDF5'
+version = '1.8.19'
+
+homepage = 'https://support.hdfgroup.org/HDF5/'
+description = """HDF5 is a unique technology suite that makes possible the management of
+ extremely large and complex data collections."""
+
+toolchain = {'name': 'foss', 'version': '2017a'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+source_urls = ['https://support.hdfgroup.org/ftp/HDF5/releases/hdf5-%(version_major_minor)s/hdf5-%(version)s/src']
+
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['7f568e2464d4ab0a74d16b23956d900b']
```

Comment:
@jhein32 Please use SHA256 checksums rather than MD5; we're moving towards making those required in PRs (see #5001)

Note that using SHA256 checksums requires EasyBuild v3.3.0 or more recent.

Worth mentioning: support for injecting/updating checksums in existing easyconfigs is in the works, see https://github.com/easybuilders/easybuild-framework/pull/2286 (hopefully to be included in EasyBuild v3.4.0)